src/app/modules/form/public-form/public-form.controller.ts

```diff
@@ -403,6 +405,7 @@ export const _handleFormAuthRedirect: ControllerHandler<
             return SgidService.createRedirectUrl(
               formId,
               Boolean(isPersistentLogin),
+              [],
```

Comment:
Note to reviewers: This will be where the MyInfo fields are passed in when we enable this in the frontend. Blank for now as we don't want to enable this feature yet.